packages/api/package.json

```
@@ -76,9 +76,9 @@
     "node": ">=18.0.0"
   },
   "dependencies": {
-    "@tbd54566975/dwn-sdk-js": "0.2.8",
     "@web5/agent": "0.2.5",
     "@web5/common": "0.2.2",
+    "@tbd54566975/dwn-sdk-js": "0.2.10",
```

Package.json sorts alphabetically
```suggestion
    "@tbd54566975/dwn-sdk-js": "0.2.10",
    "@web5/agent": "0.2.5",
    "@web5/common": "0.2.2",
```